displayname.lua

```diff
@@ -1,4 +1,4 @@
 DisplayName =
 {
-	['English'] = L"Fortships: Conquer the Skies",
+	['English'] = L"Fortships: Waky Build",
```

Comment:
@Wakkkkkkk as a learning thing, try making and pushing a commit that undoes this change.